.mailmap

```diff
@@ -33,6 +33,8 @@ Dave Abrahams <dabrahams@apple.com> <dave@Skree.local>
 Dave Abrahams <dabrahams@apple.com> <dave@Wingy.local>
 Dave Lee <davelee@lyft.com> <davelee.com@gmail.com>
 David Rönnqvist <david.ronnqvist@gmail.com> <david.ronnqvist@skype.net>
+David Rönnqvist <david.ronnqvist@gmail.com> <david.ronnqvist@gmail.com>
```

Comment:
It looks like you can just write this as `David Rönnqvist <david.ronnqvist@gmail.com>`. That makes it look less like a typo. (I had to go look up the [git-shortlog docs](https://git-scm.com/docs/git-shortlog) to see what it meant to put the same email address twice.)